tokens/src/components/Spinner.json

```diff
@@ -1,12 +1,14 @@
 {
-  "spinner": {
-    "width": { "value": "2rem", "source": "$spinner-width" },
-    "height": { "value": "{spinner.width.value}", "source": "$spinner-height" },
-    "border-width": { "value": ".25em", "source": "$spinner-border-width" },
-    "sm": {
-      "width": { "value": "1rem", "source": "$spinner-width-sm" },
-      "height": { "value": "{spinner.sm.width}", "source": "$spinner-height-sm" },
-      "border-width": { "value": ".2em", "source": "$spinner-border-width-sm" }
+  "size": {
+    "spinner": {
+      "width": { "value": "2rem", "source": "$spinner-width" },
```

Comment:
maybe we should add another key here like `default`? looks strange like this because we have `sm` key later

Reply:
done